app/scripts/models/account.js

```diff
@@ -414,10 +416,13 @@ define(function (require, exports, module) {
         }
       })
       .then((updatedSessionData) => {
+        this._metrics.logEvent('core.signin.success');
```

Comment:
Along similar lines we could also re-define signup.success and say if the user verifies and `this.isSignUp() === true`, that counts as a signup.success.